Eatery/Controllers/Eateries/EateriesSharedViewController.swift

```diff
@@ -158,6 +127,13 @@ class EateriesSharedViewController: UIViewController {
         }
     }
 
+    private func updateShowPill() {
+        let showPill = pillViewController.pillView.leftSegmentSelected && campusNavigation.viewControllers.count == 1
```

Comment:
Nit: clearly operator precedence tells how this is evaluated, but could parenthesis be added around the and clauses for nicer readability?